services/app/src/contexts/ChessBoardContext.jsx

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
+/* eslint-disable react/prop-types */
+import React, { createContext, useState } from 'react';
+import { useChessBoard } from '../hooks/useChessBoard';
+import { NUMBER_OF_ROWS, NUMBER_OF_COLUMNS } from '../constants/board';
+import '../static/style/gameScore.css';
+
+export const ChessBoardContext = createContext();
+let num;
+
+const ChessBoardContextProvider = ({ children }) => {
+  const [selectedItem, setSelectedItem] = useState('');
+  const [board, setBoard] = useChessBoard();
+
+  const placeItemOnBoard = (row, tile) => {
+    if (selectedItem) {
+      const copyOfBoard = [...board];
+      copyOfBoard[row][tile].items.push(
```

Comment:
Is the naming correct for the variable named `tile` here? I think `column` would fit better here.